step/step.go

```diff
@@ -136,24 +138,8 @@ func (step SaveCacheStep) compress(paths []string) (string, error) {
 	}
 	archivePath := filepath.Join(tempDir, fileName)
 
-	tarArgs := []string{
-		"--use-compress-program",
-		"zstd --threads=0 --long", // Use CPU count threads, enable long distance matching
-		"-P",                      // Same as --absolute-paths in BSD tar, --absolute-names in GNU tar
-		"-cf",
-		archivePath,
-		"--directory",
-		step.envRepo.Get("BITRISE_SOURCE_DIR"),
-	}
-	tarArgs = append(tarArgs, paths...)
-
-	cmd := step.commandFactory.Create("tar", tarArgs, nil)
-
-	step.logger.Debugf("$ %s", cmd.PrintableCommandArgs())
-
-	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
```

Comment:
Good call moving this to its own file. I'll do the same for the decompression code. 👍